ui_framework/src/components/header/_header.scss

```diff
@@ -0,0 +1,158 @@
+// Header. Includes breadcrumbs of nav buttons.
+
+$kuiHeaderChildSize: $kuiSizeXXL + $kuiSizeL;
+
+@include component('kuiHeader') {
+  background: $kuiHeaderBackgroundColor;
+  border-bottom: $kuiBorderThin;
+
+  @include child('logo') {
+    position: relative;
+    height: $kuiHeaderChildSize;
+    padding: $kuiSize $kuiSizeXL;
+    display: inline-block;
+    vertical-align: middle;
+
+    &:after {
+      position: absolute;
+      content: "";
+      width: 1px;
+      top: $kuiSize;
+      bottom: 0;
+      right: 0;
+      background: $kuiBorderColor;
+    }
```

Comment:
Another alternative is to use a mixin which applies the separator, but I think it makes more sense to expose it as a standalone component instead of tightly coupling it with particular components' implementations.